beacon-chain/rpc/beacon/validators_test.go

```diff
@@ -90,17 +91,37 @@ func TestServer_ListValidatorBalances_NoResults(t *testing.T) {
 	db := dbTest.SetupDB(t)
 	defer dbTest.TeardownDB(t, db)
 
+	config := &featureconfig.Flags{
+		NewStateMgmt: true,
+	}
+	featureconfig.Init(config)
```

Comment:
The same global state resetting here. There are several occurences in this file (just grep `NewStateMgmt: true`.